### ISSUE TITLE
Helper blade directives

### ISSUE THREAD

**Issue by ebrahimahmadi** [laravel/ideas#2466]
Use helper directives instead of writing them inside my blade directly. for example instead of using:

`{{ Illuminate\Support\Str::ascii('û') }}`

use something like :

`@ascii('û')`

**Comment by mfn** [comment]
You can do so already, see https://laravel.com/docs/8.x/blade#extending-blade

**Comment by ebrahimahmadi** [comment]
> You can do so already, see https://laravel.com/docs/8.x/blade#extending-blade

Yeah, I've seen that before. But I meant a set of plugins that comes across with the package that has implemented the directives fluently.
